server/src/streem/util.ts

```diff
@@ -0,0 +1,33 @@
+import { Request } from 'express';
+import { createHmac, timingSafeEqual } from 'crypto';
+import { SIGNING_KEY_SECRET } from '../routes/webhooks';
+
+export function validateWebhookRequest(req: Request): Boolean {
+    const hmac = createHmac('sha256', Buffer.from(SIGNING_KEY_SECRET, 'utf-8').toString());
+
+    // make sure the request is current
+    const sent_at = req.headers['streem-sent-at']! as string;
```

Comment:
`req.headers[...]` is of type `string | string[]`. Using `req.headers[...] || ''` doesn't resolve to `string` only; which is what I need to avoid a type conflict.